# config/settings.py
DATABASE_URI = 'sqlite:///data/dataminer.db'
LOG_FILE = 'logs/dataminer.log'
USER_AGENT = 'Mozilla/5.0 (compatible; DataMiner/1.0)'
